src/ethereum/networkAdapters/PulsechainScanAdapter.ts

```diff
@@ -146,7 +146,7 @@ export class PulsechainScanAdapter extends NetworkAdapter<PulsechainScanAdapterC
     const edgeTransaction: EdgeTransaction = {
       blockHeight,
       currencyCode,
-      date: parseInt(scanTx.timestamp),
+      date: new Date(scanTx.timestamp).getTime(),
```

Comment:
This needs to be in seconds so divide by 1000

Reply:
`EdgeTransaction['date']` is a unix timestamp in seconds?